libs/printing/src/render.tsx

```diff
@@ -61,6 +61,15 @@ function getContentHeight(page: Page): Promise<number> {
   });
 }
 
+export async function launchChromium(): Promise<Browser> {
```

Comment:
I could see an argument for leaving this with "Chromium" in the name, but I'd also be happy if this were `launchBrowser`.